ethers-providers/src/transports/common.rs

```diff
@@ -191,7 +192,7 @@ pub enum Authorization {
 
 impl Authorization {
     pub fn basic(username: impl Into<String>, password: impl Into<String>) -> Self {
-        let auth_secret = base64::encode(username.into() + ":" + &password.into());
+        let auth_secret = BASE64_STANDARD.encode(username.into() + ":" + &password.into());
```

Comment:
this looks like a separate change that bumps base64

mind submitting this separately?

Reply:
see #2032